@navikt/core/react/src/form/combobox/types.ts

```diff
@@ -31,7 +31,17 @@ export interface ComboboxProps
   extends FormFieldProps,
     Omit<
       InputHTMLAttributes<HTMLInputElement>,
-      "size" | "onChange" | "value" | "defaultValue"
+      | "size"
+      | "onChange"
+      | "value"
+      | "defaultValue"
+      | "onClick"
+      | "onInput"
+      | "type"
+      | "role"
+      | "onKeyUp"
+      | "onKeyDown"
+      | "autoComplete"
```

Comment:
Is the reason for omitting these extra attributes simply that we use our own, internal ones?
Should we `Pick<InputHTMLAttributes<HTMLInputElement>, "which" | "props" | "we" | "use">` instead? 
I guess there are still a lot of attributes left that we don't actually use? 🤔

Reply:
The reason for omitting these (the ones I added) is because we override them, so they don't work anyways. So now you will get TS error if you try to use them, instead of just silently failing.

I'm not sure how strict we should be about which props to accept. One could argue that since the input is so specialized, we should use `Pick` instead so that we know that we only offer props that works properly. This will avoid another "`placeholder` incident" where we have to say that "yes, it's supported, but not really" 😅

Reply:
I'll change to `Pick` in a new PR.